### PR TITLE
Feat authentication

### PR DESCRIPTION
**Pull request by ankit9015**
Authentication and PrivateRoute

### CODE REVIEW

**Review by ankit9015 — COMMENTED** [review]
fixed

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,39 @@
 import "./App.css";
-import logo from "./logo.png";
+import "./utils/utility.css";
+import { useState } from "react";
+import Header from "./features/Header/Header";
+import Sidebar from "./features/Sidebar/Sidebar";
+import Widgets from "./features/Widgets/Widgets";
+import Router from "./Router/Router";
```
Comment (SaurabhChawda) [inline src/App.js:7]: Use can create Index file for import and export.

**Review by SaurabhChawda — APPROVED** [review]
Overall look good to me.

**Review by MaruthiPothuganti — COMMENTED** [review]
Clean and well written code. Authentication feature works as expected. PR looks good to merge

**Pull request merged**